packages/ckeditor5-remove-format/tests/manual/removeformat.md

```diff
@@ -6,5 +6,6 @@ Select the content and press the "Remove Format" button in the toolbar.
 
 * Bold, emphasize and underline should be removed.
 * Links **must not** be removed.
+* Image **must not** change its size.
```

Comment:
Good catch that the description should be changed from "Image \*\*must not\*\* return to its original size." after changing the behavior to "Image \*\*must not\*\* change its size." 👍